pyqtgraph/parametertree/SystemSolver.py

```diff
@@ -52,16 +53,16 @@ class SystemSolver(object):
     1) The *defaultState* class attribute: This is a dict containing a 
        description of the variables in the system--their default values,
        data types, and the ways they can be constrained. The format is::
-       
+       ```
            { name: [value, type, constraint, allowed_constraints], ...}
-       
-       * *value* is the default value. May be None if it has not been specified
-         yet.
-       * *type* may be float, int, bool, np.ndarray, ...
-       * *constraint* may be None, single value, or (min, max)
-            * None indicates that the value is not constrained--it may be 
-              automatically generated if the value is requested.
-       * *allowed_constraints* is a string composed of (n)one, (f)ixed, and (r)ange. 
+       ```
+         * *value* is the default value. May be None if it has not been specified
```

Comment:
Incidentally, indenting this bulleted list makes it part of the literal environment above. I'm not sure how to delineate literal blocks with trailing indented content. I tried `.. code-block::` instead of the shorthand `::` above and added an extra line between the `{name: ...` line and the list but it still made the list part of the code block.

Reply:
I added a `Where:` on a line by itself in between the two, which should fix this.

Reply:
Perfect